packages/wallets/src/components/DerivAppsSection/DerivAppsGetAccount.tsx

```diff
@@ -96,17 +103,17 @@ const DerivAppsGetAccount: React.FC = () => {
             </div>
             <div className='wallets-deriv-apps-section__get-content'>
                 <div className='wallets-deriv-apps-section__details'>
-                    <WalletText size='sm' weight='bold'>
-                        Options
-                    </WalletText>
-                    <WalletText size={isDesktop ? '2xs' : 'xs'}>One options account for all platforms.</WalletText>
+                    <Text size='sm' weight='bold'>
+                        {localize('Options')}
```

Comment:
Isn't the general rule to use `<Localize/>` component here?

Reply:
Hi @nijil-deriv, Alright I'll update for all places.